cms/envs/common.py

```diff
@@ -158,6 +158,9 @@
 ############################ FEATURE CONFIGURATION #############################
 
 PLATFORM_NAME = _('Your Platform Name Here')
+
+CONTACT_MAILING_ADDRESS = 'SET-ME-PLEASE'
```

Comment:
Recommend this be `_('Your Contact Mailing Address Here')` - this adds i18n, and gives a more descriptive text as to what the value should be. Please make more descriptive if you'd like!

Reply:
Thanks for the suggestion. I've updated my PR.